packages/docs/src/components/header/header.css

```diff
@@ -52,7 +52,7 @@ header .mobile-menu {
 
 header .menu-toolkit {
   @apply hidden lg:flex;
-  @apply pt-0 pb-6 lg:pt-0;
+  @apply pt-0 lg:pt-0 pb-0;
```

Comment:
just out of curiosity: wouldn't we get the same result by deleting this line?

Reply:
I wanted to make as few edits as possible, so I first just removed pb-6, but then saw there was padding soming from elsewhere, so then added `pb-0`

I could've just changed `pb-6` to `pb-0` for a cleaner diff, but the change purely was to remove the bottom padding

Reply:
got it 🙏  just thought since all values are set to `0` now, we could remove the whole line since it does not affect anything. or at least i was not able to see any different behavior 👼  thanks for this adjustment 🙏 🚀